github/util.py

```diff
@@ -41,12 +41,14 @@ def __init__(
         github: GitHub,
         repository_owner: str,
         repository_name: str,
+        repository_branch: str='master',
```

Comment:
rename to 'default_branch' (optional parameter, initialised with master (as is already the case)
consider renaming repository_owner / repository_name to owner, name respectively